services/client/dev/elements/components/app-subnav/app-subnav.js

```diff
@@ -86,9 +86,13 @@ export default class AppSubnav extends Mixin(LitElement)
   }
 
   async doSelected(){
+    let schema = this.AppStateModel.location.query?.schema; // ?schema=x
```

Comment:
the schema logic is a bit too opinionated for `app-subnav`. it should live in `admin-database-subnav` component.